client/components/main/header.styl

```diff
@@ -155,3 +88,109 @@
       border-left: 1px solid rgba(255, 255, 255, .3)
       height: 24px
       float: left
+
+#header-quick-access
+  color: white
+  transition: background-color 0.4s
+  background: #2573a7
+  padding: 10px 10px
```

Comment:
Why did you enlarge the quick-access bar even on desktop (with adding 20px of vertical padding)? I would recommend to keep it unchanged on desktop.

Anyway I have some plan to transform this top bar into a left sidebar as does Rocket.chat for instance.